packages/data-point/lib/reducer-types/reducer-helpers/reducer-default/resolve.js

```diff
@@ -0,0 +1,19 @@
+const isNil = require('lodash/isNil')
+
+const utils = require('../../../utils')
+
+/**
+ * @param {Accumulator} accumulator
+ * @param {*} _default
+ * @returns {Accumulator}
+ */
+function resolve (accumulator, _default) {
+  let value = accumulator.value
+  if (Number.isNaN(value) || isNil(value) || value === '') {
```

Comment:
looking at isNil in lodash I dont think its worth depending on it, + its documentation might not be what it actually is

https://github.com/lodash/lodash/blob/912d6b04a1f6b732508a6da72a95ec4f96bda154/lodash.js#L11967-L11989

link above is TOO SLOW!!!

```js
 /**
     * Checks if `value` is `null` or `undefined`.
     *
     * @static
     * @memberOf _
     * @since 4.0.0
     * @category Lang
     * @param {*} value The value to check.
     * @returns {boolean} Returns `true` if `value` is nullish, else `false`.
     * @example
     *
     * _.isNil(null);
     * // => true
     *
     * _.isNil(void 0);
     * // => true
     *
     * _.isNil(NaN);
     * // => false
     */
    function isNil(value) {
      return value == null;
    }
```

I would recommend instead do a manual check against undefined and null

Reply:
done